src/sidebar/components/test/FilterStatus-test.js

```diff
@@ -52,20 +52,21 @@ describe('FilterStatus', () => {
   }
 
   function assertButton(wrapper, expected) {
-    const buttonProps = wrapper.find('LabeledButton').props();
+    const button = wrapper.find('Button[data-testid="clear-button"]');
+    const buttonProps = button.props();
 
     assert.equal(buttonProps.title, expected.text);
-    assert.equal(buttonProps.icon, expected.icon);
+    assert.equal(button.find('CancelIcon').exists(), !!expected.icon);
```

Comment:
The boolean coercion doesn't appear to be strictly necessary here since we control the value of `expected.icon` and are setting it to an actual boolean in all cases.